cmd/analyze.go

```diff
@@ -33,7 +33,7 @@ var analyzeCmd = &cobra.Command{
 	Short: "Analyzes an image: [image]",
 	Long:  `Analyzes an image using the specifed analyzers as indicated via flags (see documentation for available ones).`,
 	Args: func(cmd *cobra.Command, args []string) error {
-		if err := validateArgs(args, checkAnalyzeArgNum, checkArgType); err != nil {
+		if err := validateArgs(args, checkAnalyzeArgNum); err != nil {
```

Comment:
Now that we don't validate strings up front, how long does it take to fail if we pass an invalid argument?

Reply:
marginal amount of time more, if any. the validation happens before any processing happens

Reply:
Would you mind attaching the output of an error case to the PR?

Reply:
whoops didn't see this, let me generate one